### PR TITLE
chore: synchronise to ocm-api-model 0.0.365

### PR DESCRIPTION
**Pull request by Lir10**

### CODE REVIEW

**Review by gshilin-sdb — APPROVED** [review]

**Review by gshilin-sdb — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -27,7 +27,7 @@ export PATH := $(LOCAL_BIN_PATH):$(PATH)
 export CGO_ENABLED=0
 
 # Details of the model to use:
-model_version:=v0.0.364
+model_version:=v0.0.365
```
Comment (tzvatot) [inline Makefile:30]: We usually put this as bump MRs, and document in the CHANGES.md accordingly. See 0.1.410 as example

Comment (Lir10) [reply Makefile]: done

Comment (tzvatot) [reply Makefile:30]: @Lir10 are you planing to update the CHANGES.md in a followup MR?

**Review by tzvatot — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (Lir10) [reply Makefile:30]: yep here it is https://github.com/openshift-online/ocm-sdk-go/pull/936

**Review by tzvatot — APPROVED** [review]
LGTM

**Pull request merged**